### ISSUE TITLE
process is not defined in browser

### ISSUE THREAD

**Issue by eduboxgithub** [hwbllmnn/shp-write#1]
I need a lib to export shapefiles from a geojson source.
I was searching and your fork seems the most updated one, since the original from mapbox as no current maintainers.

I changed in my dependencies from the original repo to this fork:

> npm install hwbllmnn/shp-write#maintenance -save

but I'm getting an error when trying to download the zip file:

![image](https://user-images.githubusercontent.com/13800439/66708854-3c37cd00-ed4f-11e9-9a27-c8fc552a154f.png)

My project uses Angular 8 and typescript, but it seems this is using the old browserify to export for the web. As far as I know process is exclusive from node. 

from zip.js
```
return zip.generateAsync({
    type: process.browser === undefined ? 'nodebuffer' : 'blob',
    compression: 'DEFLATE'
});
```

Could this validation just verify if the window object or the process object is undefined ?

**Comment by hwbllmnn** [comment]
I guess if `process` is undefined, we can fall back to `blob` as well. Can you prepare a PR for that?

**Comment by hwbllmnn** [comment]
Fixed in #4
